src/bin/lifeblood-manager.rs

```diff
@@ -125,6 +126,13 @@ fn main() {
         panic!("failed to get current dir!");
     };
 
+    if let Some(qt_scale_factor) = option_env!("QT_SCALE_FACTOR") {
```

Comment:
`option_env!` captures env variable at **compile time**. I bet if we would want to use this env variable - we'd want to check it at runtime, instead of inheriting scale factor from building system.

But  anyway, plz check the other comment about DPI first